ext/sockets/sendrecvmsg.c

```diff
@@ -457,9 +460,12 @@ void php_socket_sendrecvmsg_init(INIT_FUNC_ARGS)
 #endif
 	REGISTER_LONG_CONSTANT("SO_PASSCRED",			SO_PASSCRED,		CONST_CS | CONST_PERSISTENT);
 #endif
-#ifdef LOCAL_CREDS_PERSISTENT
+#if defined(LOCAL_CREDS_PERSISTENT)
 	REGISTER_LONG_CONSTANT("SCM_CREDS2",			SCM_CREDS2,		CONST_CS | CONST_PERSISTENT);
 	REGISTER_LONG_CONSTANT("LOCAL_CREDS_PERSISTENT",	LOCAL_CREDS_PERSISTENT,	CONST_CS | CONST_PERSISTENT);
+#elif defined(LOCAL_CREDS)
+	REGISTER_LONG_CONSTANT("SCM_CREDS",			SCM_CREDS,		CONST_CS | CONST_PERSISTENT);
+	REGISTER_LONG_CONSTANT("LOCAL_CREDS",			LOCAL_CREDS,	        CONST_CS | CONST_PERSISTENT);
```

Comment:
LOCAL_CREDS and SCM_CREDS are also available on FreeBSD, right? Just thinking if it makes sense to make both available on FreeBSD..?

Reply:
it is but to avoid confusion I purposely did not make it available in the ancillary data context since it s not compatible with the proper layer (the one compatible with LOCAL_CREDS does not return the process id).

Reply:
ah ok that makes sense